src/containers/Search.jsx

```diff
@@ -44,9 +44,24 @@ class Search extends Component {
       });
   };
 
+  insertionsort = (inputArr) => {
```

Comment:
you should have a key function argument as the second parameter here which takes a function according to which the array is sorted by;